detect.go

```diff
@@ -97,7 +104,12 @@ func Detect(detect DetectFunc, options ...Option) {
 	}
 
 	if config.logger.IsDebugEnabled() {
-		config.logger.Debug(BuildpackPathFormatter(ctx.Buildpack.Path))
+		if internal.NewDirectoryContentsWriter(
```

Comment:
I think we are missing an err assignment here

Reply:
Yes, good catch. I'll fix.